Tests/GitHubContents.tests.ps1

```diff
@@ -335,6 +335,67 @@ try
             }
         }
 
+        Context 'When setting new file content by path' {
+            BeforeAll {
+                $filePath = 'notes'
+                $fileName = 'hello.txt'
+                $commitMessage = 'Commit Message'
+                $content = 'This is the content for test.txt'
+                $branchName = 'master'
+                $committerName = 'John Doe'
+                $committerEmail = 'john.doe@testdomain.com'
+                $authorName = 'Jane Doe'
+                $authorEmail = 'jane.doe@testdomain.com'
+
+                $contentPath = New-TemporaryFile
+                $content | Out-File -FilePath $contentPath
+
+                $setGitHubContentParms = @{
+                    Path = "$filePath/$fileName"
+                    CommitMessage = $commitMessage
+                    Branch = $branchName
+                    ContentPath = $contentPath
```

Comment:
Technically speaking, `$contentPath` at this point in time is a `[FileInfo]`, not a `[String]`.  So, what you want here is:

```suggestion
                    ContentPath = $contentPath.FullName
```